src/meta/model_v2/migration/src/m20240911_083152_variable_vnode_count.rs

```diff
@@ -21,6 +21,18 @@ impl MigrationTrait for Migration {
             )
             .await?;
 
+        manager
+            .exec_stmt(
+                UpdateStatement::new()
+                    .table(Table::Table)
+                    .values([(Table::VnodeCount, Expr::value(1))])
+                    .and_where(Expr::col(Table::DistributionKey).eq(Expr::value("[]")))
```

Comment:
Not sure if this is correct for `I32Array`. 🥵 @yezizp2012

Reply:
`I32Array` is actually a wrapped type of `json`, I'm also not sure about it. 🥵 You can test it locally.

Reply:
I've tested locally against SQLite and it works as expected. Not sure about Postgres. 🤣

Reply:
Can you do a quick test about Postgres as well? We have been tripped up by the different behaviors of sqlite and pg too many times. 🥵